Update on credential - create - delete

arbordeleau opened the pull request:
- Use Image from nextjs, add an alt atribute, updated the width of all the images (distortion) and added a default image for when no image are provided
- Updated with react-hook-form for updates on the screen when something changes
- Disabled Import credential button
- Move to react-query and updated types to use openApi types
- Implemented delete API 
- Removed the API call from the store and created a new use-credential for queries

arbordeleau committed:
Update on credential - create - delete

ronal2do commented on components/credentials/credentials-form.tsx:
```diff
@@ -1,164 +1,150 @@
-			fetchIssuers();
-		}
-	}, [mode, issuers, fetchIssuers]);
+	const onSubmit = () => {
```
```suggestion
	const onSubmit = (data: CredentialDefinitionRequest) => { 
```
onSubmit of the form, pass the data for you here, you can handle the data here, once  you don't need to use like 
```
const formData: any = form.getValues();
```

ronal2do commented:
A good practice is to use branch to reflect the ticket name, next time, avoid using a branch called "ariane-updates-v2" once when we want to search for a branch or on Jira search for a PR related to a ticket we will never be able to find it

ronal2do committed:
Merge branch 'main' into ariane-updates-v2

ronal2do committed:
refactor: Update credential and onboarding components
- Added `CredentialSchemaRequestType` to `openapi-types.ts` for improved type safety.
- Enhanced `Sidebar` component with alt text for icons to improve accessibility.
- Removed unused `CharacterEdit` and `CharacterForm` components to streamline the codebase.
- Updated `CredentialsForm` to utilize the new `CredentialSchemaRequestType` for form handling.
- Improved UI components across onboarding and scenario screens for better user experience.
